app/assets/stylesheets/static_pages.css

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
các file css, j s, helper nếu ko dùng thì xóa đi nhé
-> xóa file này

Reply:
Đoạn này em có thắc mắc là file nào nên xóa, file nào k. Vì hồi nãy em xem trong file application.css chỉ toàn nội dung được comment nên em nghĩ k quan trọng, em xóa. Lúc chạy lại thì nó báo lỗi